src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.LocalFunctions.cs

```diff
@@ -133,7 +133,15 @@ public override BoundNode VisitLocalFunctionStatement(BoundLocalFunctionStatemen
             var oldReads = usages.ReadVars;
             usages.ReadVars = BitVector.Empty;
 
-            if (!localFunc.WasCompilerGenerated) EnterParameters(localFuncSymbol.Parameters);
+            if (!localFunc.WasCompilerGenerated)
+            {
+                EnterParameters(localFuncSymbol.Parameters);
+
+                foreach (var parameter in localFuncSymbol.Parameters)
+                {
+                    _variableUsePass.Visit(parameter.ExplicitDefaultExpression);
```

Comment:
Just checking - as far as I can see, there's no way to mutate the stack depth (the "current depth" is passed in via constructor), so this would require a new visitor for every visit. Is that okay?